lte/gateway/c/oai/include/s1ap_messages_def.h

```diff
@@ -176,3 +176,43 @@ MESSAGE_DEF(
   MESSAGE_PRIORITY_MED,
   itti_s1ap_ue_context_mod_resp_fail_t,
   s1ap_ue_context_mod_failure)
+MESSAGE_DEF(
+  S1AP_ENB_CONFIGURATION_TRANSFER_LOG,
```

Comment:
What is the purpose of this "LOG" message and others below? Some of them are defined but never used.

Reply:
These 'LOG' are used as msg_ids and after decoding the pdu based on these msg_ids we can free the Pdu. I can remove which are not being used.